src/menu.rs

```diff
@@ -178,43 +233,14 @@ pub enum MenuItem {
   /// - **Windows / Android / iOS:** Unsupported
   ///
   Separator,
-}
-
-impl MenuItem {
-  /// Create new custom menu item.
-  /// unique_menu_id is the unique ID for the menu item returned in the EventLoop `Event::MenuEvent(unique_menu_id)`
-  pub fn new(title: impl ToString) -> Self {
-    let title = title.to_string();
-    MenuItem::Custom(CustomMenu {
-      id: MenuId::new(&title),
-      name: title,
-      keyboard_accelerators: None,
-    })
-  }
 
-  /// Assign keyboard shortcut to the menu action. Works only with `MenuItem::Custom`.
+  /// Represents a Submenu(label, enabled, sub menu)
```

Comment:
should this be `Submenu(title, enabled, sub menu)` or we change the `title` argument in all of our menu functions to be `label`